src/libs/actions/ReportActions.ts

```diff
@@ -33,14 +35,55 @@ function clearReportActionErrors(reportID: string, reportAction: ReportAction) {
         return;
     }
 
+    if (keys) {
+        const errors: Record<string, null> = {};
+
+        keys.forEach((key) => {
+            errors[key] = null;
+        });
+
+        Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${originalReportID}`, {
+            [reportAction.reportActionID]: {
+                errors,
+            },
+        });
+        return;
+    }
     Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${originalReportID}`, {
         [reportAction.reportActionID]: {
             errors: null,
         },
     });
 }
 
-export {
-    // eslint-disable-next-line import/prefer-default-export
-    clearReportActionErrors,
-};
+/**
+ * 
+ignore: undefined means we want to check both parent and children report actions
+ignore: parent or child means we want to ignore check parent or child report actions because we already check them before
+ */
+function clearAllRelatedReportActionErrors(reportID: string, reportAction: ReportAction | null, ignore?: IgnoreDirection, keys?: string[]) {
+    const errorKeys = keys ?? Object.keys(reportAction?.errors ?? {});
+    if (!reportAction || errorKeys.length === 0) {
+        return;
+    }
+
+    clearReportActionErrors(reportID, reportAction, keys);
+
+    const report = ReportUtils.getReport(reportID);
+    if (report?.parentReportID && report?.parentReportActionID && ignore !== 'parent') {
+        const parentReportAction = ReportActionUtils.getReportAction(report.parentReportID, report.parentReportActionID);
+        const parentErrorKeys = Object.keys(parentReportAction?.errors ?? {}).filter((err) => errorKeys.includes(err));
+
+        clearAllRelatedReportActionErrors(report.parentReportID, parentReportAction, 'child', parentErrorKeys);
```

Comment:
Please point out 1 case where we need to call `clearAllRelatedReportActionErrors` instead of `clearReportActionErrors`. 
The current logic is fine but I am not sure if we need it or not